hazelcast/src/main/java/com/hazelcast/client/impl/connection/AddressProvider.java

```diff
@@ -30,7 +31,7 @@ public interface AddressProvider {
      * @return The possible member addresses to connect to.
      * @throws Exception when a remote service can not provide addressee
      */
-    Addresses loadAddresses() throws Exception;
+    Addresses loadAddresses(ClientConnectionProcessListener listener) throws Exception;
```

Comment:
Can you update the Javadoc with the new parameter?

Reply:
Done.